mmeval/metrics/gradient_error.py

```diff
@@ -0,0 +1,176 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import cv2
+import numpy as np
+from typing import Dict, List, Sequence
+
+from mmeval.core import BaseMetric
+
+
+def gaussian(x, sigma):
+    """Gaussian function.
+
+    Args:
+        x (array_like): The independent variable.
+        sigma (float): Standard deviation of the gaussian function.
+
+    Return:
+        np.ndarray or scalar: Gaussian value of `x`.
+    """
+    return np.exp(-x**2 / (2 * sigma**2)) / (sigma * np.sqrt(2 * np.pi))
+
+
+def dgaussian(x, sigma):
+    """Gradient of gaussian.
```

Comment:
The function name and summary line do not match well.

Reply:
Yes, it has been modified in the latest update.